lib/zipper.ex

```diff
@@ -112,14 +112,14 @@ defmodule Styler.Zipper do
   def up({tree, meta}) do
     children = Enum.reverse(meta.l, [tree | meta.r])
     {parent, parent_meta} = meta.ptree
-    {make_node(parent, children), parent_meta}
+    {replace_children(parent, children), parent_meta}
   end
 
   @doc """
   Returns the zipper of the left sibling of the node at this zipper, or nil.
   """
   @spec left(zipper) :: zipper | nil
-  def left({tree, %{l: [ltree | l], r: r} = meta}), do: {ltree, %{meta | l: l, r: [tree | r || []]}}
+  def left({tree, %{l: [ltree | l], r: r} = meta}), do: {ltree, %{meta | l: l, r: [tree | r]}}
```

Comment:
Just double-checking: can `r` never be `nil`? I assume that's why this ` || []` would have been there before.